samples/common/Kconfig

```diff
@@ -73,4 +73,14 @@ config GOLIOTH_SAMPLE_WIFI_PSK
 
 endif # GOLIOTH_SAMPLE_WIFI && !GOLIOTH_SAMPLE_WIFI_SETTINGS
 
+config GOLIOTH_SAMPLE_TEST_CONNECT_TIMEOUT
+	int "Test connect timeout"
+	default 120 if NRF_MODEM_LIB
```

Comment:
The ability to store different defaults for different boards is quite powerful. This is a good tip for future use!